### ISSUE TITLE
Expiration timer stopped working for more than 59 seconds after upgrade to 3.1 

### ISSUE THREAD

**Issue by SinisterOH** [openhab/openhab-addons#10944]
## Expected Behavior
When expiration timer is set to 1 minute, it should send specified command after 1 minute.

## Current Behavior

Expiration is working only up to 59 seconds correctly, when picking 1 minute or more, nothing is happening, or expiration command is send much later (like 9,10 or 12 minutes when 1 minute is configured)

## Possible Solution

Keep it as 59 seconds until solution is provided

## Steps to Reproduce (for Bugs)

1. Create item as create point from thing in the model.
2.  Configure expiration in metadata to be more than 1 minute

## Context

Automatically turning my garage light doesn't work anymore and keeping it as 59 seconds doesn't do the job. 
It was working as expected for a longer period of time when items were configured manually, not added from model, on 3.0.1 version. I've combined both upgrade and rebuilding structure to the model, so not 100% sure what caused the problem.

Not sure if helpful, but this functionality turns off lights using KNX binding. 

## Your Environment

OH 3.1
Raspberry PI

* Version used: (e.g., openHAB and add-on versions)
* Environment name and version (e.g. Chrome 76, Java 8, Node.js 12.9, ...):
* Operating System and version (desktop or mobile, Windows 10, Raspbian Buster, ...):

**Comment by Rossko57** [comment]
> When expiration timer is set to 1 minute, it should send specified command after 1 minute.

We should qualify that,  with " ... provided no further updates arrive at the Item."
When an Item is linked to a device via a binding, the binding may issue periodic updates.
Update-to-same-value are not normally visible in your events.log
A change here may be a change in binding or device config behavior.

If you suspect a fault in expire feature, reproduce the problem using an Item not linked to a binding.

**Comment by SinisterOH** [comment]
@Rossko57 Hi,
How can I further check if item received any updates that are not visible in the standard log?
Strange this is that it was working without issues for last 3 months being used multiple times a day and now, as I upgraded to 3.1 and removed all items and recreated them from Model (add point from thing) it refuses to work again.

I created a new switch item, no connected to anything and expiration times seems to be working there correctly.
